.rubocop.yml

```diff
@@ -0,0 +1,67 @@
+AllCops:
```

Comment:
I totally agree. It's even better to avoid additional per project configuration but share exactly same set of rules.